src/test/java/net/marcellperger/mathexpr/MathSymbolTest.java

```diff
@@ -18,6 +18,17 @@ void test_fmt() {
         assertExprFmt(CommonData.getBigData2_minimumParens());
     }
 
+    @Test
+    void test_fmt__pow() {
+        // TODO decide spacing: `2.2**3.1` or `2.2 ** 3.1`
```

Comment:
```
TODO decide spacing: `2.2**3.1` or `2.2 ** 3.1`
```

Reply:
| `2.2**3.1` | `2.2 ** 3.1` |
|--------|--------|
| This is how I'd do the spacing in Python | The status quo |
| More compact, less ugly IMO | Consistent with other operators | 
| Show the precedence better | |

Reply:
So probably no spaces so TODO change this